frontend/src/layouts/Admin.css

```diff
@@ -0,0 +1,4 @@
+logo {
+  height: 2rem;
+  width: 2rem;
+}
```

Comment:
This seems to restrict the logo to be a square? Have we tried with other logos that are rectangular?

Reply:
This appeared to be the constraint on the old placeholder logo. I have used rectangular logos, and it doesn't change the size of the header.